snippets/config/app/permissions.xml

```diff
@@ -4,10 +4,10 @@
         ...
     </meta>
     <permissions>
-        <read>product</read>
-        <create>product</create>
-        <update>product</update>
+        <!-- CRUD shortcut - equivalent to read, create, update, delete for product (since 6.7.3.0) -->
+        <crud>product</crud>
```

Comment:
Would like to add a code block instead where we show both, otherwise the usage of single operations will be lost

![2025-08-27_15-08](https://github.com/user-attachments/assets/d192dce7-3dc4-455a-95a6-f8228ca460f2)

I will add the syntax and code later if this is :ok: